### DISCUSSION TITLE
Please tell me your favorite application!

### DISCUSSION THREAD

**Discussion by wyx19981124** [NAIST-SE/AttractingContributors#572] [General]
The smartphone is one of the indispensable parts of our life.
Furthermore, the app is considered the basic component.
I would like to learn about many other apps that I haven't known.
Please tell me your favorite apps here:
https://github.com/wyx19981124/AttractingContributors-app
In addition, it is welcome to comment your url if needed.

**Comment by XelaAl** [comment]
I can't choose only one. I like DuoLingo, Evernote, Cool Reader, Podcast Addict, and Momento.

**Comment by McJul** [comment]
Not favorite yet, because I'm still testing it. Came across with the reviews of JustAnswer app at [https://justanswer.pissedconsumer.com/customer-service.html](https://justanswer.pissedconsumer.com/customer-service.html), and decided to try. It's something new and interesting for me. So I think, it'll be interesting for you too.
